### ISSUE TITLE
Doesn't understand PASS in filter column

### ISSUE THREAD

**Issue by keiranmraine** [bihealth/vcfpy#129]
* vcfpy version: 0.12.0
* Python version: 3.6.5
* Operating System: Ubuntu Bionic

### Description

PASS is a valid value for the filter column and shouldn't be required in header:

http://www.internationalgenome.org/wiki/Analysis/vcf4.0/

### What I Did

```
UnknownFilter: Filter not found in header: PASS; problem in FILTER column
  UnknownFilter,
```

Thanks

**Comment by holtgrewe** [comment]
Whoops, good catch! Will look into this next week.

**Comment by holtgrewe** [comment]
Fixed in `develop`, will make new point release after CI passes.

**Comment by holtgrewe** [comment]
Fixed in `v0.12.1`. Thanks for reporting and sorry for the delay (`vcfpy` was issuing a warning which should not have affected the running code).
